docs/modules/druid/partials/nav.adoc

```diff
@@ -18,3 +16,9 @@
 *** xref:druid:usage-guide/operations/pod-placement.adoc[]
 *** xref:druid:usage-guide/operations/pod-disruptions.adoc[]
 *** xref:druid:usage-guide/operations/graceful-shutdown.adoc[]
+* xref:druid:reference/index.adoc[]
+** xref:druid:reference/crds.adoc[]
+*** {crd-docs}/druid.stackable.tech/druidcluster/v1alpha1/[DruidCluster]
```

Comment:
Hard-coding `v1alpha1` can introduce problems in the future once we start versioning CRDs. E.g. druid-op 24.x can handle DruidCluster v1alpha1, v1beta1, `v1 and v2. We would avoid the problem by only linking to https://crds.stackable.tech/druid-operator/nightly/

Reply:
I'd say we leave this for now? Everything is versioned. So we can just change the link once we change our CRD versions too. This link will always be valid in the future.

Reply:
I would really like to link direclty to the DruidCluster. Once we version things, I'd always link to the stable/most recent version of the CRD.

Reply:
Or maybe once we have a lot of version we stop linking to the object, but at least for now I think it's quite useful and I'd leave it in

Reply:
Works for me